docs/developer-guide/joins/join-streams-and-tables.md

```diff
@@ -296,6 +295,14 @@ table, if the corresponding key exists already in the join result table.
 | 14        |                  | d                |                  |                  | [null, d]        |
 | 15        | D                |                  | [D, d]           | [D, d]           | [D, d]           |
 
+Foreign-Key Joins
+-----------------
+
+ksqlDB also supports 1:N foreign-key joins.
+Many-to-many (N:M) joins are not supported currently.
+For a foreign-key join, you can use any left input-table column in the join condition
+to join it with the primary key of the right input-table.
+
```

Comment:
Should we add a subsection called `Semantics of Foreign-Key Table-Table Joins`, in order to parallel the other sections above?